src/main/java/org/fogbowcloud/manager/core/constants/ConfigurationConstants.java

```diff
@@ -40,4 +40,7 @@
     public static final String REVERSE_TUNNEL_PRIVATE_ADDRESS_KEY = "reverse_tunnel_private_address";
     public static final String REVERSE_TUNNEL_PORT_KEY = "reverse_tunnel_port";
     public static final String REVERSE_TUNNEL_HTTP_PORT_KEY = "reverse_tunnel_http_port";
+
+    // HISTORY & RECOVERY DATABASE CONF
+    public static final String DATABASE_URL = "history_recovery_database_url";
```

Comment:
Thinking as a fogbow user, I do not think that this property name is representative.

Reply:
Okay, I will change it to `manager_jdbc_url` as we discussed.

Reply:
Thank you Viga :top:! This name solves the misunderstand problem with others databases urls :smile:.

Reply:
Exactly, thank you too, hehe :man_technologist:.

Reply:
Call it jdbc_manager_url instead.

Reply:
@fubica `jdbc_manager_url` can be understood as the manager url in jdbc.

Reply:
I see the point now. At first, I didn't understand that manager was a reference to fogbow manager. I wonder if we need to have manager in the variable name. Do we have other jdbc_url in this config file? Perhaps we could just call it jdbc_url.

Reply:
In the manager I don't think so, but in others services with databases, such as the federated network service, certainly there will be a `jdbc_url` constant, but these constants will be separated by conf files.

Reply:
I think the name could be just `jdbc_database_url`, since it represents the database url defined by the jdbc patterns.